Parsers/ASimUserManagement/Parsers/vimUserManagementCiscoISE.yaml

```diff
@@ -0,0 +1,175 @@
+Parser:
+  Title: User Management ASIM filtering parser for Cisco ISE
+  Version: '0.1'
+  LastUpdated: July 13, 2023
+Product:
+  Name: Cisco ISE
+Normalization:
+  Schema: UserManagement
+  Version: '0.1.1'
+References:
+- Title: ASIM User Management Schema
+  Link: https://aka.ms/ASimUserManagementDoc
+- Title: ASIM
+  Link: https:/aka.ms/AboutASIM
+- Title: Cisco ISE Security Events 
+  Link: https://www.cisco.com/c/en/us/td/docs/security/ise/3-2/admin_guide/b_ise_admin_3_2/b_ISE_admin_32_maintain_monitor.html#ID58
+Description: |
+  This ASIM parser supports normalizing user management activity in the Cisco ISE events to the ASIM User Management schema.
+ParserName: ASimUserManagementCiscoISE
+EquivalentBuiltInParser: _Im_UserManagement_CiscoISE
+ParserParams:
+  - Name: starttime
+    Type: datetime
+    Default: datetime(null)
+  - Name: endtime
+    Type: datetime
+    Default: datetime(null)
+  - Name: srcipaddr_has_any_prefix
+    Type: dynamic
+    Default: dynamic([])
+  - Name: actorusername_has_any
+    Type: dynamic
+    Default: dynamic([])
+  - Name: targetusername_has_any
+    Type: dynamic
+    Default: dynamic([])
+  - Name: eventtype_in
+    Type: dynamic
+    Default: dynamic([])
+  - Name: eventresult
+    Type: string
+    Default: '*'
+  - Name: disabled
+    Type: bool
+    Default: false
+ParserQuery: |
+    let EventFieldsLookup=datatable(
+    EventOriginalType: int,
+    EventResult: string,
+    EventResultDetails: string,
+    EventType: string,
```

Comment:
seems like 'EventResultDetails' and 'EventType' values are interchanged. Can you pls check and fix.
https://learn.microsoft.com/en-us/azure/sentinel/normalization-schema-user-management
Also recheck 'EventSubType' values
Please apply the same change to vim parser as well.

Reply:
rest things looks good. Thankyou

Reply:
Seems like lookup table is correct
Some of  `EventType` and `EventSubType` values are interchanged because in ASIMUserManagement Schema reference we have a limited available options for `EventSubType`:
`UserRead`: Password, Hash;  `UserCreated`, `GroupCreated`, `UserModified`, `GroupModified`
hence, EventTypes like **UserCreated** has the same `EventSubType` value
`EventResultDetails` values are populated for **Failure** events only, for `EventResult == 'Success' `they are blank
If our vision is not correct, please show example what exactly must be changed and we will do that

Reply:
Please refer the below screenshot, this is the output I get after running this parser:
![image](https://github.com/Azure/Azure-Sentinel/assets/97222872/d6214281-2b69-400a-bb88-4fedd1d5e18b)

Reply:
Thanks for the clarification. Fixed